### ISSUE TITLE
IPv6 localAddress results in Error: bind EINVAL xxx

### ISSUE THREAD

**Issue by rala-l** [apify/proxy-chain#219]
If I set the localAddress  to a Ipv4 it works. 
**If I set the localAddress  to a IpV6 then Error: bind EINVAL.** 

Both Ip`s are available on NodeJS host.  (Windows 10, NPM 8.6.0, Node v16.14.2, proxy-chain 2.0.1)

I tried direct modification on source code and the same result.

src/direct.ts
    const options = {
        port: '443', 
        host: 'google.com',  
        localAddress: 'db8: 3333 : 4444 : 5555 : 6666 : 7777 : 8888' 
    };
....
    const targetSocket = **net.createConnectio**n(options, () => {
...
Results in  
„bind EINVAL 2001 : db8: 3333 : 4444 : 5555 : 6666 : 7777 : 8888” (fake IP)

**Comment by szmarczak** [comment]
The IPv6 address you provided is invalid.

**Comment by thomas510111** [comment]
This is a legitimate issue https://stackoverflow.com/questions/11199776/how-do-i-bind-local-ipv6-address-to-my-node-js-program

**Comment by szmarczak** [comment]
You need to provide `family` option as well. Example:

```js
http.get('http://www.example.com', { localAddress: 'xxxx:xxxx:xxxx:xxx:xxxx:xxxx:xxx:xxxx', family: 6 }, res => res.pipe(process.stdout)); '';
```

**Comment by szmarczak** [comment]
I think you solved it in #224
